velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -1420,6 +1436,13 @@ class PlanBuilder {
       const std::vector<std::string>& windowFunctions,
       bool inputSorted);
 
+  PlanBuilder& topNRowNumberBase(
```

Comment:
Move this one to private scope, and I'm also not sure if we should have 3 functions to interpret the nodes.

Reply:
This is already in private space.